packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -67,6 +67,34 @@ QUnit.module('basics', {}, () => {
         assert.equal($element.find('.' + INPUT_CLASS).attr('inputmode'), 'decimal', 'inputmode is correct');
     });
 
+    QUnit.module('Tabindex', () => {
+        const testCases = [
+            { initial: false, updated: true, expectedInitial: -1, expectedUpdated: 0 },
```

Comment:
initial what? That's not ok to name a variable with an adjective only, it's difficult to read now